src/server.js

```diff
@@ -12,7 +12,7 @@ container.register("dbUrl", "mongodb://127.0.0.1:27017/test");
 container.register("JWT_SECRET", process.env.JWT_SECRET || "adsfasdf123adf");
 container.register("TOKEN_LIFE_TIME",  process.env.TOKEN_LIFE_TIME || "1500");
 container.register("userRoute", {});
-container.register("messageRoute", {});
+//container.register("messageRoute", {});
```

Comment:
You can just remove this

Reply:
well i only left it there because i didnt know what it really did, it just created a problem for me and i decided to comment it out and see if fixes things

Reply:
okay removed